source/clear-linux/guides/maintenance/download-image.rst

```diff
@@ -0,0 +1,192 @@
+.. _download-image:
+
+Download, verify, and uncompress a Clear Linux image
+####################################################
+
+.. _types-of-cl-images:
+
+Types of Clear Linux images
+===========================
+
+|CL| offers many types of images for different platforms and environments.
+
+The naming convention of a |CL| image filename is: 
+
+  .. code-block:: console
+   
+    clear-<version number>-<image type>.<compression type>
+
+The `<version number>` field specifies the version number.
+
+The `<image type>` field specifies the type of image and its corresponding file format.  The current list of images available are:
```

Comment:
This list of image types could become outdated quickly.  I wonder if we should make this a separate file that we can update independently